### ISSUE TITLE
Missing UI and unit tests

### ISSUE THREAD

**Issue by jasonnam** [jorelosorio/JorelTest#4]

**Comment by jorelosorio** [comment]
Hi, so the idea is to complete the UI and unit test or do you will introduce a bug in those fields? :) Because I did a simple UI test but not unit tests.

**Comment by jasonnam** [comment]
Seems like the test case is not belong to unit tests.
UI tests usually boot up simulator and manipulate the app while the app is visually opened.

**Comment by jorelosorio** [comment]
Yes, you are right but for this case, I tested only the view controllers
with the *view models* filling view with dummy models, to make sure all the
view are built correctly. I could make the views to run the app and
simulate a user is interacting with the app, but if the number of surveys
changes in the request that the test could fail. I separated the view so
the build could be tested with fake data.
